x-pack/test/security_solution_api_integration/test_suites/edr_workflows/artifacts/trial_license_complete_tier/index.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { getRegistryUrl as getRegistryUrlFromIngest } from '@kbn/fleet-plugin/server';
+import { isServerlessKibanaFlavor } from '@kbn/security-solution-plugin/scripts/endpoint/common/stack_services';
+import { FtrProviderContext } from '../../../../ftr_provider_context_edr_workflows';
+import { ROLE } from '../../../../config/services/security_solution_edr_workflows_roles_users';
+import {
+  getRegistryUrlFromTestEnv,
+  isRegistryEnabled,
+} from '../../../../../common/services/security_solution';
+
+export default function endpointAPIIntegrationTests(providerContext: FtrProviderContext) {
+  const { loadTestFile, getService } = providerContext;
+
+  describe('Endpoint plugin', function () {
+    const ingestManager = getService('ingestManager');
+    const rolesUsersProvider = getService('rolesUsersProvider');
+    const kbnClient = getService('kibanaServer');
+    const log = getService('log');
+
+    if (!isRegistryEnabled()) {
```

Comment:
I can see that we are importing `isRegistryEnabled` and `getRegistryUrlFromTestEnv` straight away without `getService` hooks.

This is my current implementation. It required the functions to be called in a before or after scope. @elastic/platform-qx , have you got any recommendation for calling `getService` v.s. import the services?

https://github.com/elastic/kibana/pull/187257/files#diff-acafb17ea5b60d25d069f4c6821cfae362b4a3908d6e64612411fdda1d998190R24-R31

Reply:
I think you pinged the wrong team, good luck however

Reply:
Sorry about that 🙈

Reply:
https://elastic.slack.com/archives/C5UDAFZQU/p1720690934928169

Reply:
@szwarckonrad I'll leave the decision to you about using import v.s. getService. I'll update my PR according to your decision.